packages/cozy-client/src/types.js

```diff
@@ -92,6 +92,7 @@ import { QueryDefinition } from './queries/dsl'
  * @property {object} metadata - Metadata of the file
  * @property {object} type - Type of the file
  * @property {object} class - Class of the file
+ * @property {object} relationships - Relationships of the file
```

Comment:
Not sure this is needed : 
- 1/ IOCozyFile should extends CozyClientDocument
- 2/ relationships are optional

Reply:
Sans ça y'a une erreur de lint. Je vais regarder si ça extend bien CozyClientDocument, ou si on peut indiquer que c'est optionnel

Reply:
C'est modifié, j'ai extend CozyClientDocument